src/rendergraph/common/rendergraph/assert.h

```diff
@@ -0,0 +1,3 @@
+// rendergraph is a module but we want to include util/assert.h from Mixxx,
+// without adding the entire src/ dir to the include paths.
+#include "../../../util/assert.h"
```

Comment:
Not a fan of the relative import from the parent dir. We don't do that anywhere else. Not sure if there is a better way.

Reply:
It’s a bit ugly, but I didn’t want to add the entire src/ dir to the include paths.

Reply:
I'm wondering if we could get CMake to copy `assert.h` into the rendergraph include? CMake noob so I have no idea if it could work

Reply:
I really wouldn't go down that path. This is isolated and commented.